src/frontends/onnx/frontend/src/core/graph.cpp

```diff
@@ -76,7 +76,7 @@ Graph::Graph(const std::shared_ptr<ONNX_NAMESPACE::ModelProto>& model_proto,
                 // invalid external data makes initializers creation impossible
                 throw;
             } catch (const ngraph::ngraph_error&) {
-                ng_constant = default_opset::Constant::create(tensor.get_ng_type(), Shape{}, {0});
+                ng_constant = ngraph::onnx_import::common::make_failsafe_constant(tensor.get_ng_type());
```

Comment:
Maybe we can change namespaces? 😄

Reply:
Those 2 new helpers were added to an existing namespace in common.hpp. When it's time to refactor the whole ONNX FE, it will be easier to spot and rename all occurrences of `ngraph::` symbols.

Reply:
Yes I just think, maybe we need to plan this activity.